Tools/build/deepfreeze.py

```diff
@@ -402,7 +417,7 @@ def generate(self, name: str, obj: object) -> str:
             # print(f"Cache hit {key!r:.40}: {self.cache[key]!r:.40}")
             return self.cache[key]
         self.misses += 1
-        if isinstance(obj, (types.CodeType, umarshal.Code)) :
+        if isinstance(obj, types.CodeType) :
```

Comment:
here `umarshal.Code` won't work, because `generate` requires a lot more attrs that `umarshal.Code` has